backend/services/ephemeris_service.py

```
@@ -62,34 +99,74 @@ def get_current_positions(self, lat: Optional[float] = None, lon: Optional[float
+                    degree = lon_deg % 30
+                    
+                    # Extract speed for retrograde calculation
+                    speed = result[0][3] if isinstance(result[0], (tuple, list)) and len(result[0]) > 3 else 1.0
+                    
+                    positions[name] = {
+                        "sign": ZODIAC_SIGNS[sign_index],
+                        "degree": round(degree, 2),
+                        "longitude": round(lon_deg, 2),
+                        "retrograde": speed < 0
+                    }
+                except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)